raiden-ts/tests/unit/mocks.ts

```diff
@@ -1,21 +1,32 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-
-import { patchEthersDefineReadOnly, patchMatrixGetNetwork } from './patches';
+import { patchEthersDefineReadOnly, patchEthersGetNetwork, patchVerifyMessage } from './patches';
+patchVerifyMessage();
 patchEthersDefineReadOnly();
-patchMatrixGetNetwork();
+patchEthersGetNetwork();
 
-import { AsyncSubject, of, BehaviorSubject, Subject } from 'rxjs';
+import { AsyncSubject, of, BehaviorSubject, ReplaySubject } from 'rxjs';
 import { MatrixClient } from 'matrix-js-sdk';
 import { EventEmitter } from 'events';
 import { memoize } from 'lodash';
 import logging from 'loglevel';
+import { Store, createStore, applyMiddleware } from 'redux';
 
+// TODO: remove this mock
```

Comment:
Is there some reason blocking us from removing it right now?

Reply:
`jest.mock` returns a mocked instance of the classes in the module. We use it on the old `raidenEpicDeps`, and need to keep it there while there're tests using these mocks. You can see the new mocks use `jest.requireActual` to get the actual class, and just mock/spy on certain methods. After we refactor the remaining tests, and can remove the old mocks and revert mocking the whole providers mock.

Reply:
Ok that makes sense, thank you.